lib/public/views/QcFlags/QcFlagsModel.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { QcFlagsForDataPassOverviewModel } from './ForDataPass/QcFlagsForDataPassOverviewModel.js';
+import { QcFlagsForSimulationPassOverviewModel } from './ForSimulationPass/QcFlagsForSimulationPassOverviewModel.js';
+import { Observable } from '/js/src/index.js';
+
+/**
+ * Quality Control Flags model
+ */
+export class QcFlagsModel extends Observable {
+    /**
+     * The constructor of the model
+     */
+    constructor() {
+        super();
+
+        this._forDataPassOverviewModel = new QcFlagsForDataPassOverviewModel();
+        this._forDataPassOverviewModel.bubbleTo(this);
+
+        this._forSimulationPassOverviewModel = new QcFlagsForSimulationPassOverviewModel();
+        this._forSimulationPassOverviewModel.bubbleTo(this);
+    }
+
+    /**
+     * Load the overview page model
+     *
+     * @returns {void}
+     */
+    loadForDataPassOverview({ dataPassId, runNumber, dplDetectorId }) {
+        this._forDataPassOverviewModel.runNumber = parseInt(runNumber, 10);
+        this._forDataPassOverviewModel.dataProductionId = parseInt(dataPassId, 10);
```

Comment:
For me this is too much factorization, and here you should name the parameter dataPassId

Reply:
refactored